packages/migrate-config/README.md

```diff
@@ -28,6 +28,13 @@ bunx @eslint/migrate-config .eslintrc.json
 
 The tool will automatically find your `.eslintignore` file in the same directory and migrate that into your new configuration file.
 
+If you receive any npm errors while running using `npx`, install the package manually first:
+
+```shell
+npm i @eslint/migrate-config
+npx @eslint-migrate-config .eslintrc.json
```

Comment:
made a few changes:
1. `-g`
2. add `@latest` to avoid caching issues.
```suggestion
npm i @eslint/migrate-config@latest -g
eslint-migrate-config .eslintrc.json
```

Reply:
I'm not sure we want to have people install globally that's why I'm still using `npx`.

Reply:
I was trying to avoid `npx` globally installing, which is where the problems can occur. In this case, `npx` just runs locally.